Umbraco.AuthTokens/Data/UmbracoAuthTokenFactory.cs

```diff
@@ -18,20 +18,26 @@ public static UmbracoAuthToken GenerateUserAuthToken(UmbracoAuthToken authToken)
         {
             //Date Time
             var dateCreated = DateTime.UtcNow;
+           
             var dateCreatedToString = dateCreated.ToString("u");
+            //this probably should be allowed to be configurable but keeping it simple and reasonable for now
+            var dateExpires = dateCreated.Add(TimeSpan.FromMinutes(60));
```

Comment:
Yes for now this is fine @codenakama 
But I agree with your comment that it should be configurable in the future as you may not want to have a token with an hour expiry and may wish to refresh/get a new token more often